gcp/appengine/frontend3/src/templates/vulnerability.html

```diff
@@ -17,6 +17,14 @@ <h1 class="title">
           <a class="vulnerability-improvement-link" href="{{ vulnerability.human_source_link }}/improve">
             Suggest an improvement
           </a>
+        {% elif vulnerability.human_source_link -%}
+          <div>
+          <a class="vulnerability-improvement-link" href="https://google.github.io/osv.dev/faq/#ive-found-something-wrong-with-the-data" title="Follow the Source link below and use its record feedback reporting mechanism">
+            See a problem?
+          </a>
+          <br/>
+          Please try reporting it to <a class="vulnerability-improvement-link" href="{{ vulnerability.human_source_link}}" target="_blank" rel="noopener noreferrer">the source</a> first.
```

Comment:
We might want to remove the class from this as this is part of a sentence it looks a bit weird with the big monospaced font.

Reply:
Removed and relocated per your other comment in 1cb455b5